nerdtree_plugin/fs_menu.vim

```diff
@@ -388,44 +388,67 @@ endfunction
 
 " FUNCTION: NERDTreeQuickLook() {{{1
 function! NERDTreeQuickLook()
-    let treenode = g:NERDTreeFileNode.GetSelected()
-    if treenode !=# {}
-        call system("qlmanage -p 2>/dev/null '" . treenode.path.str() . "'")
```

Comment:
Most of these will allow injection when operating on a file that looks like...

```
'; touch FOOBAR ; echo '
```
Opening this file on Linux with `mo`, and the file `FOOBAR` will appear in Vim's current working directory.  This snippet provides a simplified example of the mechanism:

```vim
func! g:Foo()
    return "'; touch FOOBAR ; echo '" 
endfunc

call system("xdg-open '" . g:Foo() . "' &")
```

This problem seems like it would never happen, but you can do some funky things in the terminal... like concealing characters that actually are contained in the path using terminal escape codes.  It's definitely better to use `shellescape()`.